dojo/finding/views.py

```diff
@@ -1133,6 +1145,11 @@ def apply_template_to_finding(request, fid, tid):
             finding.last_reviewed = timezone.now()
             finding.last_reviewed_by = request.user
             finding.tags = form.cleaned_data['tags']
+
+            finding.cve = None
+            Vulnerability_Reference.objects.filter(finding=finding).delete()
+            finding_helper.save_vulnerability_references(finding, form.cleaned_data['vulnerability_references'].split())
```

Comment:
I prefer to have the line 1150 in the the `save_vulnerability_references()` method.
you choose to create a method to manage the vulnerability_references attribute.

Reply:
This `delete` is not necessary here. The `save_vulnerability_references()` method handles the old Vulnerability References anyway. I will remove this line.